eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -251,9 +251,12 @@ jobs:
               path: $(Build.ArtifactStagingDirectory)/packages
 
       # Generate Documentation
-      - pwsh: >-
-          choco install doxygen.install
-        displayName: Install Doxygen
+      - pwsh: |
+          $setupFile="doxygen-1.8.15-setup.exe";
+          Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.15-setup.exe" `
+          -OutFile $setupFile | Wait-Process; Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT")
```

Comment:
The `Start-Process` should probably include a `-Wait` because subsequent steps are failing: 
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=302263&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=22be136e-4813-5a04-5aa2-6a647ac29f4d&l=37
```
FileNotFoundError: [WinError 2] The system cannot find the file specified
```